examples/with-zones/now.json

```diff
@@ -6,8 +6,13 @@
     { "src": "home/next.config.js", "use": "@now/next" }
   ],
   "routes": [
-    { "src": "/blog/_next(.*)", "dest": "blog/_next$1" },
-    { "src": "/blog", "dest": "blog/blog" },
-    { "src": "(.*)", "dest": "home$1" }
-  ]
+    { "src": "/blog/(_next|static)(.*)", "dest": "blog/$1$2" },
```

Comment:
```suggestion
    { "src": "/blog/(_next|static)(.*)" },
```

Reply:
Wow, this works! Wouldn't have guessed so.